app/services/pii/password_encryptor.rb

```diff
@@ -6,7 +6,7 @@ def initialize
     end
 
     def encrypt(plaintext, user_access_key)
-      encrypted_key_maker.make(user_access_key)
+      encrypted_key_maker.make(user_access_key) unless user_access_key.made?
```

Comment:
So my understanding is the issue is that Rails auto reloading will make a new class named `UserAccessKey` when the file changes, but the old files in memory will be instances of the old class, so that's why the `.kind_of?` check fails.

However in production this is never really going to happen, so this seems like a lot of changes to accomodate a weird quirk of development in Rails?

Would it be easier to duck-type? Ex have `key_maker` check for methods it wants on `user_access_key` rather than the type? It's a little more Ruby/interface-friendly IMO. It would also be useful if for some reason in a test we wanted to stub/mock a `UserAccessKey` and weren't able to make a real one?